util/ci/check_png_optimized.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+optimized=0
+
+for file in $(git ls-files *.png); do
+	 # Does only run a fuzzy check without -o7 -zm1-9 since it would be too slow otherwise
+	output=$(optipng -nc -strip all -simulate $file 2>&1 >/dev/null)
+	if ! grep -q "is already optimized" <<< $output; then
```

Comment:
missing quoting

Reply:
I don't think it matters in this case because of `<<<`, but I changed it anyway.